www/js/main.js

```diff
@@ -3124,19 +3129,28 @@ function overlayMap( callback ) {
 function debugWU() {
 	var popup = "<div data-role='popup' id='debugWU' class='ui-content ui-page-theme-a'><table class='debugWU'>";
 
-	if ( weather ) {
-		popup += "<tr><td>" + _( "Humidity" ) + "</td><td>" + weather.humidity + "%</td></tr>" +
-			"<tr><td>" + _( "Mean Temp" ) + "</td><td>" + formatTemp( weather.temp ) + "</td></tr>" +
-			"<tr><td>" + _( "Precip Today" ) + "</td><td>" + formatPrecip( weather.precip ) + "</td></tr>" +
-			"<tr><td>" + _( "Adjustment Method" ) + "</td><td>" + getAdjustmentMethod( controller.options.uwt ).name + "</td></tr>" +
-			"<tr><td>" + _( "Current % Watering" ) + "</td><td>" + controller.options.wl + "%</td></tr>";
-	}
-
-	popup += ( typeof controller.settings.lwc === "number" ? "<tr><td>" + _( "Last Weather Call" ) + "</td><td>" + dateToString( new Date( controller.settings.lwc * 1000 ) ) + "</td></tr>" : "" ) +
-				( typeof controller.settings.lswc === "number" ? "<tr><td>" + _( "Last Successful Weather Call" ) + "</td><td>" + dateToString( new Date( controller.settings.lswc * 1000 ) ) + "</td></tr>" : "" ) +
```

Comment:
Suggest re-title as "Weather Server Last Seen" to avoid user confusion when an api call between Firmware and App connects but an error is returned.

Reply:
Just to be clear you mean change `Last Successful Weather Call` to `Weather Server Last Seen` correct?

Reply:
What about 'Last Weather Call Received', or 'Last Weather Response Received'. Basically I assume Pete's point is to avoid the term 'Successful' as it can be misunderstood.

Reply:
yeah, `lswc` holds the timestamp for what is more like the last network connection between Firmware and Weather Service. This is a bit different to what an average user would consider a "successful call". In essence, if the weather service returns a non-zero `errCode` then lswc would be set but the user would probably consider it "unsuccessful" given there wasn't a valid water scale update. So just suggesting a fine-tune of the language to ensure the user is clear.

Reply:
Okay updated this per both of your suggestions, thank you!